package Helper;

import java.sql.Connection;
import java.sql.SQLException;

import javax.naming.Context;
import javax.naming.InitialContext;
import javax.naming.NamingException;
import javax.sql.DataSource;

import java.sql.*;

public class ConnectionHelper {
	static Connection conn = null;
public Connection createConnection(){
	
		try {
			Context context=new InitialContext();
			DataSource ds=(DataSource)context.lookup("java:comp/emp/jdbc/EmployeeDB");
			conn= ds.getConnection();
		} catch (NamingException e) {
			e.printStackTrace();
		} catch (SQLException e) {
			// TODO Auto-generated catch block
			e.printStackTrace();
		}
	return conn;	
}
public void closeConnection(Connection conn,PreparedStatement ps,ResultSet rs)
{
	try {
		if(rs!=null)
		{
			rs.close();
			conn.close();
			ps.close();
		}
	} catch (SQLException e) {
		e.printStackTrace();
	}
}
}
